src/DotNetWorker/Invocation/DefaultFunctionExecutor.cs

```diff
@@ -1,20 +1,93 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System;
 using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.Azure.Functions.Worker.Pipeline;
 
 namespace Microsoft.Azure.Functions.Worker.Invocation
 {
     internal class DefaultFunctionExecutor : IFunctionExecutor
     {
+        private const string FunctionReturnType = "FunctionsWorkerReturnReserved";
```

Comment:
This is the reserved type for setting the binding from the `return` directly. Might want to come up with a better name.

Reply:
Discussed offline but potentially better to reuse WebJobs reserved return word, "$return"? Maybe not though there's a lot of magic around $return!!